appinventor/buildserver/build.xml

```diff
@@ -206,6 +206,35 @@
     </java>
   </target>
 
+  <!-- =====================================================================
+       PlayAppAab: builds ../build/buildserver/AppInventorCompanion.aab
+       ===================================================================== -->
+  <!-- TODO: Need to modify the CheckPlayApp and PlayApp.uptodate tasks to ensure the AAB is
+             checked and not the APK file. -->
+  <target name="PlayAppAab"
```

Comment:
I'm not sure if this is the correct approach. I'm actually duplicating quite a lot of lines of code, so maybe it could be a better idea to use `ant PlayApp --aab` or something like that.

Reply:
Humm, it's a possibility. But in that case, maybe the BuildServer URL should be changed, so instead of accepting only "aab=1/0", maybe it should accept "ext=apk/aab/ipa", to keep consistency. Should I change both of them?